bpmn-model/pom.xml

```diff
@@ -33,6 +33,10 @@
       <groupId>org.camunda.bpm.model</groupId>
       <artifactId>camunda-xml-model</artifactId>
     </dependency>
+    <dependency>
+      <groupId>io.camunda</groupId>
+      <artifactId>zeebe-util</artifactId>
+    </dependency>
```

Comment:
❌ We should not add this dependency. The Zeebe Util is not built for Java 8 and would push a lot of new transitive dependencies to consumers of the bpmn-model library. 

>**Note** it seems only to be needed for the Tuple class. Perhaps you can find a different solution without this class.